Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM busybox:1.28
+FROM busybox:1.32
```

Comment:
If nobody objects, let's use alpine here

Reply:
Will change in the next commit.